scripts/install_python_prod_dependencies.py

```diff
@@ -163,7 +166,21 @@ def _get_requirements_file_contents():
     return requirements_contents
 
 
-def _get_third_party_python_libs_directory_contents():
+def _dist_has_meta_data(
```

Comment:
Moved this to a separate function as we cannot type the arguments in a lambda function.